graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/kotlin/KotlinDataTypeGenerator.kt

```diff
@@ -101,7 +101,16 @@ class KotlinInputTypeGenerator(config: CodeGenConfig, document: Document) :
         when (value) {
             is BooleanValue -> CodeBlock.of("%L", value.isValue)
             is IntValue -> CodeBlock.of("%L", value.value)
-            is StringValue -> CodeBlock.of("%S", value.value)
+//            TODO: Test null
+            is StringValue -> {
+                val customTypeOverrides = mapOf(
```

Comment:
Would be good to add the customTypeOverrides as a util function that is invoked here.

Reply:
Actually, for clarity, let's make it is specific for now and add a `checkAndGetLocaleValue()` as a  private  function in InputTypeGenerator that looks to see if it is `java.util.Locale` and returns the language tag if so, else null.